lib/src/main/kotlin/com/swmansion/starknet/data/types/TransactionReceipt.kt

```diff
@@ -107,6 +108,9 @@ data class InvokeTransactionReceipt private constructor(
     @SerialName("finality_status")
     override val finalityStatus: TransactionFinalityStatus,
 
+    @SerialName("failure_reason")
+    override val failureReason: String? = null,
+
```

Comment:
Correct me if I'm wrong, but from what I gather from the spec `failure_reason` should only be available in `starknet_getTransactionStatus` and `starknet_getMessagesStatus` responses, not receipts.

Reply:
> Correct me if I'm wrong, but from what I gather from the spec failure_reason should only be available in `starknet_getTransactionStatus`

Yes, you're absolutely correct. It was added to receipt classes by mistake 😅 . 

> and `starknet_getMessagesStatus` responses

I don't see that `failure_reason` should be included in `starknet_getMessagesStatus`? https://github.com/starkware-libs/starknet-specs/blob/v0.8.0/api/starknet_api_openrpc.json#L266-L290